NEWS.md

```diff
@@ -2,8 +2,12 @@
 
 ## Breaking changes
 
+* `AppDriver$get_screenshot(selector=)`, `AppDriver$expect_screenshot(selector=)`'s default `selector` value changed from the HTML DOM selector `"html"` to `"scroll"`. `"scroll"` is a custom `{shinytest2}` selector that will take a screenshot of the entire scrollable area. While a breaking change, this new default value is more intuitive and robust to non-standard CSS height and width values that fail with `"html"`. (#325)
```

Comment:
It feels like `"scroll"` may not be a super intuitive name for users, but I can't think of a much better name (maybe `"contents"`, `"document"`, or `"scrollable_area"`?)

Anyway, I don't feel super strongly about, especially since it's a "hidden" default.

Reply:
... same as Winston

Reply:
Because it is a silent default, `"scrollable_area"` makes a lot of sense.  **changing